Classes/Controller/AbstractController.php

```diff
@@ -24,6 +25,8 @@
 	private static $context;
 	private $contextBE;
 
+	public $settingsOrg;
```

Comment:
Ok, at the moment it's not used and if needed in the future we could write a function getSettingOrg or something like that to gain access. Will be removed...